gradlew

```diff
@@ -114,6 +109,7 @@ fi
 if $cygwin ; then
     APP_HOME=`cygpath --path --mixed "$APP_HOME"`
     CLASSPATH=`cygpath --path --mixed "$CLASSPATH"`
+    JAVACMD=`cygpath --unix "$JAVACMD"`
```

Comment:
Why this change? This has nothing to do with the Long-thing

Reply:
This was an automatic change, I haven't touched that and can be freely discarded